os/include/tinyara/mpu.h

```diff
@@ -49,4 +49,27 @@ enum {
 #define STACKGUARD_MPU_REGION_NUM		3	/* Region number 3 is reserved for stack protection */
 #endif
 
+/* Enum for MPU region numbers */
+
+enum MPU_REG_NUM {
+#ifndef CONFIG_APP_BINARY_SEPARATION
```

Comment:
if`n`def is difficult to see. Let's do ifdef first
```
#ifdef CONFIG_APP_B...
#else 
#endif 
```

Reply:
done